WordPress/Sniffs/Theme/NoCDNSniff.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Restricts the use of the CDN URLs.
+ *
+ * @link https://make.wordpress.org/themes/handbook/review/required/
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_NoCDNSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'no_cdn';
+
+	/**
+	 * Array of functions with positions.
+	 *
+	 * The number represents the position in the function call
+	 * passed variables, here the capability is to be listed.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var array
+	 */
+	protected $target_functions = array(
```

Comment:
This is a good point. I have not seen many themes use external sites to load their images. Normally the reason being that sites do not like hotlinking.

If we wanted to check for this then we would need to check the `<img src="">` tag.